### ISSUE TITLE
Problems with regr.penalized.ridge and regr.penalized.lasso 

### ISSUE THREAD

**Issue by berndbischl** [mlr-org/mlr#612]
First of all, the NAMESPACE is again broken of that package.
The learners only work if we load the package into the global NAMESPACE.
We only dont see this, because in our unit tests we already load the package into globalenv....

So we need to add 
    package = "!penalized"
to the learner.

Second, it seems that there is a bug as both learners seem to produce the same output.

**Comment by berndbischl** [comment]
We should also by default set trace=FALSE, it really sucks

**Comment by berndbischl** [comment]
And the reason that the weka team reported similar outputs is because we not lambda1 and lambda2 correctlty in the defaults!

**Comment by berndbischl** [comment]
Actually I am stupid. The reason both learners return the same model, because the default is lamda = 0, lambda is the penalty control constant. so with lam = 0, we fit a linear model for lasso and ridge. so everything is ok.

But I do see that some other params are missing description objects. 

Can these be added. Flo would you do that?

So TODOS

1) Change trace = FALSE (and add note)
2) load package with !
3) add other params and look at docs for this

**Comment by florianfendt** [comment]
will do!

**Comment by berndbischl** [comment]
Sorry, to simply paste unfinished stuff here, this is what i could do before time ran out

```
     par.set = makeParamSet(
       makeNumericLearnerParam(id = "lambda1", default = 0, lower = 0),
+      makeUntypedLearnerParam(id = "unpenalized"),
+      makeLogicalVectorParam(id = "positive"),
+      makeNumericVectorLearnerParam(id = "startbeta"),
+      makeNumericVectorLearnerParam(id = "startgamma"),
+      makeLogicalLearnerParam(id = "standardize", default = FALSE),
+      makeIntegerLearnerParam(id = "steps", lower = 1L, default = 1L),
+      makeNumericLearnerParam(id = "epsilon", lower = 0, default = 1e-10),
+      makeIntegerLearnerParam(id = "maxiter", lower = 1L),
       makeLogicalLearnerParam(id = "trace", default = FALSE, tunable = FALSE)
+
-    note = ""
+    note = "trace=FALSE was set by default to disable logging output"
   )
 }
 
diff --git a/R/RLearner_regr_penalized_ridge.R b/R/RLearner_regr_penalized_ridge.R
index f04d511..da602db 100644
--- a/R/RLearner_regr_penalized_ridge.R
+++ b/R/RLearner_regr_penalized_ridge.R
@@ -10,7 +10,7 @@ makeRLearner.regr.penalized.ridge = function() {
     properties = c("numerics", "factors"),
     name = "Penalized Ridge Regression",
     short.name = "ridge",
-    note = ""
+    note = "trace=FALSE was set by default to disable logging output"
   )
 }
 ```

I guess we can also allow poisson type to enable count data modelling

**Comment by florianfendt** [comment]
@berndbischl
(Just checking so I don't miss stuff here) 
You dropped the fusedl param on purpose, because we would then need lambda2 in penalized.lasso and hence allowing ridge penalization in the lasso learner if fusedl = FALSE and lambda not 0.
Is that right?
If so, we could still set up a new learner regr.penalized.fusedlasso, no?

**Comment by berndbischl** [comment]
this is done, the PR was merged #850
